### ISSUE TITLE
Which user permissions are necessary for the plugin?

### ISSUE THREAD

**Issue by JakobWaegerle** [mattermost/mattermost-plugin-jira#944]
Is it necessary for the user to be a jira agent or is being able to create a new request in the Jira Customer Portal enough? Our users are being synced via AD group but only have the possibility to "connect", not "Create Jira Issue" or "Attach to Jira Issue".

**Comment by mickmister** [comment]
Hi @JakobWaegerle, you'll need to connect your account to Jira before those other two actions are available. I'm not sure if Jira Customer Portal in general is supported in this plugin.

**Comment by JakobWaegerle** [comment]
Even after connecting the options are not available. I fear it's necessary for the users to be Jira "agents" as well which would make the necessary licences go up from a handful of people to the whole company if we wanted to use that feature.

**Comment by mickmister** [comment]
So after the user is connected, do they still see "Connect to Jira" in the post action dropdown, or just not see anything related to the Jira plugin? That shouldn't ever be the case (showing no items there option), regardless of their Jira permissions.

It's possible there's an error happening here, but that doesn't seem like the case. You are sure the connection process succeeds?

**Comment by JakobWaegerle** [comment]
Yes they still see "Connect to Jira" after the connection process succeeds.

**Comment by mickmister** [comment]
@JakobWaegerle Can you please provide the output of running the command `/jira info`, run by the affected user?

**Comment by JakobWaegerle** [comment]
***@***.***

Okay it seems like she didn’t connect. However when we type in /jira connect it opens up a window with the help center where she is already logged in…

Freundliche Grüße / Kind regards

Jakob Wägerle
IT-Systemadministrator

Tel: 07071/56544380
-------------------------------------------------------------------------------
CeGaT GmbH | Paul-Ehrlich-Str. 23 | 72076 Tübingen, Germany
***@***.******@***.***> | www.cegat.de<http://www.cegat.de/>
Court District Stuttgart - HRB 729958 | VAT-No.: DE265504070
Managing Directors: Dr. Dirk Biskup, Dr. Dr. Saskia Biskup, Dr. Detlef Schumann

Von: Michael Kochell ***@***.***>
Gesendet: Dienstag, 23. Mai 2023 15:56
An: mattermost/mattermost-plugin-jira ***@***.***>
Cc: Jakob Wägerle ***@***.***>; Mention ***@***.***>
Betreff: Re: [mattermost/mattermost-plugin-jira] Which user permissions are necessary for the plugin? (Issue #944)


@JakobWaegerle<https://github.com/JakobWaegerle> Can you please provide the output of running the command /jira info, run by the affected user?

**Comment by mickmister** [comment]
>  However when we type in /jira connect it opens up a window with the help center where she is already logged in

@JakobWaegerle Is there anything related to connecting the account to Mattermost on this page that opens?

I assume by "help center" this is referring to Jira Service Management. I'm thinking that the plugin is not able to integrate with JSM.

**Comment by JakobWaegerle** [comment]
No the page that opens looks like this:
***@***.***
(took this screenshot from my device, not the users but it looks the same) and that is also what I am referring to as „help center“

Freundliche Grüße

Jakob Wägerle
IT-Systemadministrator

Tel: 07071/56544380
-------------------------------------------------------------------------------
CeGaT GmbH | Paul-Ehrlich-Str. 23 | 72076 Tübingen, Germany
***@***.******@***.***> | www.cegat.de<http://www.cegat.de/>
Court District Stuttgart - HRB 729958 | VAT-No.: DE265504070
Managing Directors: Dr. Dirk Biskup, Dr. Dr. Saskia Biskup, Dr. Detlef Schumann

Von: Michael Kochell ***@***.***>
Gesendet: Mittwoch, 24. Mai 2023 16:37
An: mattermost/mattermost-plugin-jira ***@***.***>
Cc: Jakob Wägerle ***@***.***>; Mention ***@***.***>
Betreff: Re: [mattermost/mattermost-plugin-jira] Which user permissions are necessary for the plugin? (Issue #944)


However when we type in /jira connect it opens up a window with the help center where she is already logged in

@JakobWaegerle<https://github.com/JakobWaegerle> Is there anything related to connecting the account to Mattermost on this page that opens?

I assume by "help center" this is referring to Jira Service Management. I'm thinking that the plugin is not able to integrate with JSM.

**Comment by mickmister** [comment]
@JakobWaegerle I'm not able to see your screenshot. I think you may need to paste the screenshot into GitHub's UI rather than email

**Comment by JakobWaegerle** [comment]
![image](https://github.com/mattermost/mattermost-plugin-jira/assets/133861775/3586e083-d793-42bf-9ebc-b16d2ecd4ca1)

**Comment by mickmister** [comment]
@JakobWaegerle Interesting that this is the page the user is sent to.

Unfortunately, this plugin does not support integrating with Jira Service Management. I apologize about this. Closing the issue as "won't do".
